MovieApp/LoginViewController.swift

```diff
@@ -0,0 +1,217 @@
+import Foundation
+import UIKit
+import PureLayout
+
+// https://stackoverflow.com/questions/25367502/create-space-at-the-beginning-of-a-uitextfield
+class PaddedTextField: UITextField {
+
+    let padding = UIEdgeInsets(top: 12, left: 16, bottom: 12, right: 16)
+
+    override open func textRect(forBounds bounds: CGRect) -> CGRect {
+        return bounds.inset(by: padding)
+    }
+
+    override open func placeholderRect(forBounds bounds: CGRect) -> CGRect {
+        return bounds.inset(by: padding)
+    }
+
+    override open func editingRect(forBounds bounds: CGRect) -> CGRect {
+        return bounds.inset(by: padding)
+    }
+}
+
+class LogInViewController: UIViewController{
+    
+    private let viewBackgroundColor = UIColor(red: 19/255, green: 59/255, blue: 99/255, alpha: 1)
+    private let inputBackgroundColor = UIColor(red:21/255, green:77/255, blue:133/255, alpha:1)
+    private let buttonBackgroundColor = UIColor(red:76/255, green: 178/255, blue:223/255, alpha:1)
+        
+    private var scrollView: UIScrollView!
+    private var contentView: UIView!
+    
+    private var signInLabel: UILabel!
+    
+    private var emailForm: UIView!
+    private var emailLabel: UILabel!
+    private var emailInput: PaddedTextField!
+    
+    private var passwordForm: UIView!
+    private var passwordLabel: UILabel!
+    private var passwordInput: PaddedTextField!
+    
+    private var signInButton: UIButton!
+    
+    override func viewDidLoad() {
+        
+        super.viewDidLoad()
+        buildViews()
+    }
+    
+    private func buildViews(){
+        
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+    
+    private func createViews(){
+        
+        // https://stackoverflow.com/questions/24126678/close-ios-keyboard-by-touching-anywhere-using-swift
+        let tap = UITapGestureRecognizer(target: self, action: #selector(UIInputViewController.dismissKeyboard))
+        //tap.cancelsTouchesInView = false
+        view.addGestureRecognizer(tap)
+        
+        //create scroll view
+        scrollView = UIScrollView()
+        scrollView.bounces = true
```

Comment:
`createViews` method should just instantiate a view (UI element) and add it as a subview. Nothing more. If you need to style a view, add text color, font, etc, use `styleViews` method. Fix throughout.